src/tests/intg/conftest.py

```diff
@@ -0,0 +1 @@
+from files_ops import passwd_ops_setup, group_ops_setup  # noqa
```

Comment:
Does this file exist?

Reply:
Yes. https://github.com/SSSD/sssd/blob/master/src/tests/intg/files_ops.py

Reply:
Ideally, the fixtures would not be imported but defined in conftest.py directly, but that would require some code shifting and refactoring, not worth the effort since all intg tests are going away once we migrate them to new framework.